docs/projections-and-scalebars.md

```diff
@@ -0,0 +1,28 @@
+# Projections, Measurement, and Scalebars
+
+GeoMoose uses the EPSG:3857 projection for displaying maps.
+This choice was made to ensure that users could use OpenStreetMap,
+Bing, and other popular basemap layers. This is the same projection
+that is used by other popular web mapping applications.
+
+Problematically, this Mercator projection can show a number of artifacts
+in daily use.
+
+## Measurements
+
+Measuring between two points on the screen will not necessarily yield
+an accurate measurement.  As the map goes north to south measuring points
+will yield decreasing accuracy.  GeoMoose accounts for this fact and will
+perform the necessary re-projection in order to provide the user with
+meaningful numbers.
+
+## Scalebars
+
+The scalebars included with the demo are rendered by MapServer.  MapServer
```

Comment:
I'm not sure about my square pixel argument (although I don't see yet why that would be different from 4326).   I think it particular issue is more related to mapserver assuming the X,Y of the map coordinates are in whatever the UNITS is set to (METERS for us) rather than running it through proj4 like it does for the map data.